src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+
+  if (days <= 2) {
+    return days * price;
+  }
+
+  if (days <= 6) {
+    return days * price - 20;
+  }
+
+  return days * price - 50;
```

Comment:
Consider using constants for the daily price, the discount thresholds, and the discount amounts to make your code more readable and maintainable. For example, instead of using '40', '20', and '50' directly in the calculations, define constants like DAILY_PRICE, SHORT_TERM_DISCOUNT, LONG_TERM_DISCOUNT, SHORT_TERM_THRESHOLD, and LONG_TERM_THRESHOLD at the beginning of your function.